howto/images/use-specific-release.md

```diff
@@ -0,0 +1,16 @@
+(howto-use-specific-release)=
+# How to use a specific release
+
+With every new Anbox Cloud release, updated images are published. By default, the latest image release is pulled by AMS, but you can request a specific release with the following syntax:
+
+    amc image add <local image name> <remote image name>@<release>
+
+For instance, to fetch the arm64 Android 13 image of the 1.18.0 release:
+
+    amc image add foobar jammy:android13:arm64@1.18.0
```

Comment:
(AFAICT this is taken directly from the previous manage images page - if you don't think my comment is warranted as it's not directly related to your changes, feel free to ignore it).

I think we should point to a newer version here (e.g. 1.24.1)

Also, I think that doesn't really work this way anymore, as I think we now need to specify the commit hash with the version, e.g.: `jammy:android13:arm64@1.24.1-<hash>`

@morphis